.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
-* @kdonovan @lizziekoshelev @rkcrisafi @ebabian
```

Comment:
@adrianbautista Did you mean to make this PR in shared-ui instead of teamshares/shoelace?

Reply:
making it in all 3 (slight variations in each depending on where design system related code might live):
- https://github.com/teamshares/shared-rails-engine/pull/263
- https://github.com/teamshares/shoelace/pull/20

Reply:
Ok, makes sense. Github is complaining that the file contains errors because it doesn't know about that new user, though

Reply:
oops, needed to explicitly add this github team to the repo, should be valid now thanks for the catch